pytorch_lightning/metrics/functional/precision_recall.py

```diff
@@ -444,6 +468,18 @@ def precision_recall(
         - If ``average in ['none', None]``, they are a tensor of shape ``(C, )``, where ``C`` stands for
           the number of classes
 
+    Raises:
+        ValueError:
+            If ``average`` is not one of ``"micro"``, ``"macro"``, ``"weighted"``,
+            ``"samples"``, ``"none"`` or ``None``.
+        ValueError:
+            If ``mdmc_average`` is not one of ``None``, ``"samplewise"``, ``"global"``.
+        ValueError:
+            If ``average`` is set but ``num_classes`` is not provided.
+        ValueError:
+            If ``num_classes`` is set
+            and ``ignore_index`` is not in the range ``0`` <= ``ignore_index`` < ``num_classes``.
```

Comment:
```suggestion
            and ``ignore_index`` is not in the range ``[0, num_classes)``.
```
can you change this everywhere?

Reply:
@dipam7 Applied this to other places for you.